tools/filter_compounds/filter_compounds.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="filter_orgmet_anorg" name="Filter organometallics and/or anorganics" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
+    <description>from a library of compounds</description>
+    <!--parallelism method="multi" split_inputs="infile" split_mode="to_size" split_size="50000" shared_inputs="" merge_outputs="outfile"></parallelism-->
+    <macros>
+        <import>macros.xml</import>
+        <token name="@GALAXY_VERSION@">1</token>
```

Comment:
The common practice is to start with `0` for initial release of Galaxy tool. Please consider adopting it.

Reply:
I will changed that, sorry my bad, did not notice it starts from 0.